src/astro/databases/base.py

```diff
@@ -1,13 +1,13 @@
 from abc import ABC
-from typing import Optional
+from typing import List, Optional
 
 import pandas as pd
 import sqlalchemy
 from airflow.hooks.base import BaseHook
 
 from astro.constants import (
     DEFAULT_CHUNK_SIZE,
-    AppendConflictStrategy,
+    DBMergeConflictStrategy,
```

Comment:
Why was the previous Strategy replaced? It contained all the cases for append and merge.